sdk/identity/azure-identity/azure/identity/_credentials/browser.py

```diff
@@ -52,7 +54,14 @@ class InteractiveBrowserCredential(InteractiveCredential):
 
     def __init__(self, **kwargs):
         # type: (**Any) -> None
-        self._redirect_uri = kwargs.pop("redirect_uri", None)
+        redirect_uri = kwargs.pop("redirect_uri", None)
+        if redirect_uri:
+            self._parsed_url = urlparse(redirect_uri)
+            if not (self._parsed_url.hostname and self._parsed_url.port):
+                raise ValueError('"redirect_uri" must be a URL with port number, for example "http://localhost:8400"')
```

Comment:
Should there be a field added to the docstring documenting this error? Something like:
```
:raises ValueError: if `redirect_uri` is not a URL with a port number
```